allure-support/src/main/kotlin/com/kaspersky/components/alluresupport/files/resources/impl/DefaultAllureResourcesRootDirsProvider.kt

```diff
@@ -0,0 +1,14 @@
+package com.kaspersky.components.alluresupport.files.resources.impl
+
+import com.kaspersky.components.alluresupport.files.resources.AllureResourcesRootDirsProvider
+import io.qameta.allure.kotlin.util.PropertiesUtils
+import java.io.File
+
+class DefaultAllureResourcesRootDirsProvider : AllureResourcesRootDirsProvider {
+    override val logcatRootDir = File("logcat")
+    override val screenshotsRootDir = File("screenshots")
+    override val videoRootDir = File("video")
+    override val viewHierarchy = File("view_hierarchy")
+    override val allureRootDir = File(PropertiesUtils.resultsDirectoryPath)
```

Comment:
Since the `allure.properties` can't be changed in runtime maybe we could cache the `PropertiesUtils.resultsDirectoryPath`'s result and access the cached value in `MoveReportsInterceptor` to not load it on each call?